scripts/e2e-test.sh

```diff
@@ -41,7 +43,8 @@ Options:
   --tests <list of tests>   Lists of tests to run, delimited by spaces (default: "$tests")
         Note: the last 2 tests should be (if they are to be run)
              node_disconnect/replica_pod_remove uninstall
-  --extended                Run long running tests also.
+  --profile <extended|continuous|ondemand>
+                            Run the tests corresponding to the profile.
```

Comment:
Add `(default :ondemand)` to help text

Reply:
The default is no longer equivalent to ondemand.

Reply:
Then it is unclear what happens if no profile is set?

Reply:
Like the current behaviour of e2e-test.sh ?
I could change the text to "Run the tests corresponding to the profile instead of the default tests."

Reply:
Yeah that would work.